src/protos/flywheels.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto3";
+
+/*************************************************/
+/******** Main messages that will be sent ********/
+/*************************************************/
+
+message FlywheelConfig {
+    MotorControlMethod controlMethod = 1;
+    DshotModes dshotMode = 2;
+    repeated int32 controlPins = 3;
+    TelemMethod telemetryMethod = 4;
+    // Both no telemetry and bidirectional Dshot will not have any telemetry pins
+    // Serial and tachometer will likely only have one pin, but we could support having them communicate on multiple pins
+    repeated int32 telemPins = 5;
+    int32 motorKv = 6;
+}
+
+message ClosedLoopFlywheelParams {
+    int32 revRpm = 1;
+    int32 idleRpm = 2;
+    int32 idleTime_ms = 4;
+
+    // Should this be rpm? Percent of rev rpm?
+    int32 firingThresholdRpm = 5;
+
+    // Can we get some sort of unit on this?
+    int32 spindownSpeed = 6;
+}
+
+message OpenLoopFlywheelParams {
+    int32 revDutyCycle = 1;
+    int32 idleDutyCycle = 2;
+    int32 idleTime_ms = 3;
+
+    // The minimum amount of time that the flywheels must have revved before the pusher can be triggered
+    int32 minimumRevTime_ms = 4;
```

Comment:
I think we need two parameters, maybe called spinupTime_ms and idleSpinupTime_ms, since brushless motors are significantly faster to spin up from idle than from a standstill.

Reply:
Ack